frontend/wallet/src/render/Apollo.re

```diff
@@ -33,14 +33,43 @@ let client = {
 };
 
 module Decoders = {
-  let int64 = pk =>
-    Js.Json.decodeString(pk) |> Option.getExn |> Int64.of_string;
+  [@bs.val] [@bs.scope "window"] external isFaker: bool = "";
```

Comment:
qq - I get that this is telling bucklescript to get `isFaker` from window, but don't understand why you have to add an empty string literal as the body?  Is that just convention?

Reply:
It's short for `external isFaker: bool = "isFaker";`. The first `isFaker` is the name of the ocaml/reason function you're defining and the second `"isFaker"` is the name of the JS (or if you're writing a native external, C) function you're binding to.

Reply:
got it, thx